### DISCUSSION TITLE
systeminformer version 3.0.6550 can not be compiled

### DISCUSSION THREAD

**Discussion by ElectricDigger** [winsiderss/systeminformer#1719] [Questions / Answers] [answered]
Hello, its my first question here at GitHub, hope that I do it right :-)
Because I have troubles finding the actual 64bit version of systeminformer, I started building it my self
(git clone --branch=master https://github.com/winsiderss/systeminformer.git C:\projects\systeminformer),
but I stepped into this problem, (Win10, 64bit, Visual Studio 2022 community):
![grafik](https://user-images.githubusercontent.com/55180614/236646260-f32ed3c3-5bbf-415d-9d53-54275f01b2d3.png)
Is that compilation problem known? 
Of cause I tried also building it via Visual Studio GUI (*.sln) trying to build a 32 or 64bit debug/relese version. Same problem occured.
Thanks in advance for any help.
Kind regards

**Comment by ElectricDigger** [comment]
Ok, version systeminformer 3.0.6505 comes with a 64bit version which was not removed/changed by Win10/Viruscheck. So the first question can be ignored. Is the compilation Problem known (2. question)?
Thanks and kind regards!

**Comment by dmex** [comment, marked as answer]
You're missing the Windows SDK and you need to install the latest version.

**Reply by ElectricDigger** [reply]
dmex, you are my hero!!!!
Yes this one was missing:
![grafik](https://user-images.githubusercontent.com/55180614/236650517-13990f05-04da-4147-876d-eca6f59fb608.png)
Nightly build 3.0.11121.6594 compiled and runs fine!
Thank you so much!
Kind regards from the other side of the world (Germany;-)!

**Comment by dmex** [comment]
> I have troubles finding the actual 64bit version of systeminformer

The amd64 folder contains the 64bit binaries?
